app/segments/drill/[id]/submission/input.js

```diff
@@ -187,29 +190,39 @@ function fillRandomShotTargets(min, max, drillInfo) {
   return shots;
 }
 
+//Helper function for the putt drill type
+function fillPuttTargets(drillInfo) {
+  let shots = [];
+  for (var i = 0; i < drillInfo.reps; i++) {
+    var baseline = lookUpExpectedPutts(drillInfo.requirements[0].items[i]);
+    shots.push({
+      shotNum: i + 1,
+      target: [
+        drillInfo.requirements[0].items[i],
```

Comment:
consider changing requirements to a map so we don't rely on the array being in a certain order